src/executor/current_thread/mod.rs

```diff
@@ -304,10 +311,16 @@ impl<P: Park> CurrentThread<P> {
     pub fn new_with_park(park: P) -> Self {
         let unpark = park.unpark();
 
+        // XXX: Should this use a bounded channel with an arbitrary limit instead, and then
+        // panic spawning if the channel is full?
+        let (spawn_sender, spawn_receiver) = mpsc::unbounded();
```

Comment:
Unbounded is fine here. Generally, `spawn` fns do not have back pressure. Instead, they either succeed or fail. One accepted pattern is for the executor to have a max # of tasks and `spawn` to fail if that is reached. This feature can be punted to another PR though.

Reply:
Removed the comment